pkg/apis/fission.io/v1/validation.go

```diff
@@ -304,6 +304,11 @@ func (spec FunctionSpec) Validate() error {
 		result = multierror.Append(result, spec.InvokeStrategy.Validate())
 	}
 
+	// TODO Add below validation warning
+	/*if spec.FunctionTimeout <= 0 {
```

Comment:
`spec.FunctionTimeout <= 0` -> spec.FunctionTimeout < 0

`Should always be more than 0` -> `Should always be greater than or equal to 0`